flocker/control/script.py

```diff
@@ -0,0 +1,42 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Script for starting control service server.
+"""
+
+from twisted.python.usage import Options
+from twisted.internet.endpoints import TCP4ServerEndpoint
+
+from .httpapi import create_api_service
+from ..common.script import (
+    flocker_standard_options, FlockerScriptRunner, main_for_service)
+
+
+@flocker_standard_options
+class ControlOptions(Options):
+    """
+    Command line options for ``flocker-control`` cluster management process.
+    """
+    optParameters = [
+        # XXX also choose where configuration will be stored
```

Comment:
Okay ... maybe all this new stuff (ie, the port and the path to the stored cluster configuration) **should** be in the control service process configuration file?  It's not a big deal - but I don't see a good reason to have /etc/flocker/config.yml for some configuration and command line options for other configuration.

I guess one difference is that the volume manager uuid is generated and written whereas everything else is going to be a static default value or user-supplied values... But the control service doesn't have a volume manager uuid.